tests/__snapshots__/dsl.test.ts.snap

```diff
@@ -360,7 +360,7 @@ exports[`DSL checkDSL() semantics should identify correct error line number if t
     "severity": 8,
     "source": "linter",
     "startColumn": 22,
-    "startLineNumber": 5,
+    "startLineNumber": 4,
```

Comment:
If we look at dsl.test.ts line 164 (https://github.com/openfga/syntax-transformer/blob/f2b3cc0a0fc53531e4d00b1a55484c32eeb46151/tests/dsl.test.ts#L164), the error should be identified in line 5 instead of line 4.

Reply:
@adriantam That's because of the empty line dropped by the indentDsl function.
This function helps the parser, so getting rid of empty lines is work done from the parsing side.